scripts/.storybook/main.ts

```diff
@@ -10,22 +10,40 @@ function getAbsolutePath(value: string): string {
   return dirname(require.resolve(join(value, 'package.json')));
 }
 const config: StorybookConfig = {
+  staticDirs: ['./public'],
   stories: ['./docs/**/*.stories.mdx', '../../packages/**/*.stories.@(ts|md)x'],
```

Comment:
we don't have the docs directory so we can keep this like: ` stories: ['../../packages/**/*.stories.@(ts|md)x']`